media-libs/speexdsp/speexdsp-1.2~rc3.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A patent-free, Open Source/Free Software DSP library"
+DESCRIPTION="Speex is an open-source/free software  patent-free audio \
+compression format designed for speech. The Speex project aims to lower the \
+barrier of entry for voice applications by providing a free alternative to \
+expensive proprietary speech codecs. Moreover, Speex is well-adapted to \
+internet applications and provides useful features that are not present in \
+most other codecs.
+Finally, Speex is part of the GNU project and is available under the revised \
+BSD license."
+HOMEPAGE="https://www.speex.org/"
+COPYRIGHT="2002-2009 Xiph.org Foundation
+	2002-2008 Jean-Marc Valin
+	2005-2007 Analog Devices Inc.
+	2005-2008 Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+	1993, 2002, 2006 David Rowe
+	2003 EpicGames
+	1992-1994 Jutta Degener, Carsten Bormann"
+LICENSE="Speex"
+REVISION="1"
+SOURCE_URI="https://downloads.xiph.org/releases/speex/speexdsp-${portVersion/\~/}.tar.gz"
+CHECKSUM_SHA256="4ae688600039f5d224bdf2e222d2fbde65608447e4c2f681585e4dca6df692f1"
+SOURCE_DIR="speexdsp-${portVersion/\~/}"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
SpeexDSP builds fine on all architectures :)
Shall we mark it as tested everywhere?
```diff
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="x86"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```
(Marking `x86_gcc2` as tested in `SECONDARY_ARCHITECTURES` might not be seen as needed, but it could turn out to be nice to have it if/when x86_gcc2 becomes available as a secondary arch for x86_64.)

Reply:
If you prefer this:
```
ARCHITECTURES="x86_gcc2 ?x86 x86_64"
SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
```
then that's also okay.